src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -1596,6 +1597,16 @@ public Territory getCurrentBattleTerritory() {
   public IBattle getCurrentBattle() {
     return m_currentBattle;
   }
+
+  protected void raids() {
+    // Remove air raids - this adds all the bombing raids to the battle tracker
+    for( final Territory t : m_battleTracker.getPendingSBRSites() ) {
```

Comment:
This logic is in the wrong spot. We are calling methods on battle tracker to get data from it, to then call more methods sending the same data back. Seems like this logic would be simpler/better in BattleTracker.